requirements/dev.txt

```diff
@@ -1,9 +1,10 @@
 #
```

Comment:
This file doesn't seem to have any relevant changes. Let's remove it from the PR please.